### PR TITLE
chore(codeclimate): add test script

### PR DESCRIPTION
**Pull request by ishimweeli**
### What does this PR do?
This PR is for maintaining the project's code coverage using codeclimate

### How should this be manually tested?
click on the badges in the README file

### What are the relevant pivotal tracker stories?
[#182201521](https://www.pivotaltracker.com/n/projects/2569104/stories/182201521)

### CODE REVIEW

**Comment by kalisaNkevin** [comment]
rebase this PR

**Review by kalisaNkevin — APPROVED** [review]

**Review by munyanezaarmel — APPROVED** [review]

File: .github/workflows/githubActions.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
```
Comment (ericus123) [inline .github/workflows/githubActions.yml:14]: why choose 12 version choose at least 14/16

**Comment by ericus123** [comment]
@ishimweeli where is this branch based from ???
Take a look at the changes we currently have in `dev` your branch is totally different.

**Review by ericus123 — APPROVED** [review]

File: .github/workflows/githubActions.yml (hunk 1 above)
Comment (ericus123) [inline .github/workflows/githubActions.yml]: why choose 12 version choose at least 14/16

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: .github/workflows/githubActions.yml (hunk 1 above)
Comment (ishimweeli) [reply .github/workflows/githubActions.yml]: got  it

**Comment by ericus123** [comment]
@ishimweeli rebase

**Review by ericus123 — APPROVED** [review]

**Pull request merged**